docs/tutorials/get-started/servo-mousemover.md

```diff
@@ -0,0 +1,280 @@
+
+- For "pin": use "12" - this is the pin you attached the PWM (Pulse Width Modulation) jumper wire to.
+- For "board": use "local"
+- For **Depends On** click **local.** This ensures that the board is initialized before the servo.
+
+The attribute section will look like this:
+
+![This is a screenshot of the Viam app on the CONFIG page. This shows the options that populate when you create a servo with the pi Model. It shows attributes to fill out, with a JSON coding area next to it. "Pin" is filled in with "12" and "Board" is filled in with "local". Pin and board are required fields. To the right of this area is an "Attribute Guide" with optional attributes and required. Required attributes are denoted with a star before the word. To the left below the Attributes is Data Capture Configuration and to the right of this is a button that says Add Method. Below that and to the left it says Frame and to the right of Frame it has a button that says Add Frame. Below this and to the left it says Depends On and to the right of that it has a dropdown box. Below the dropdown box you can see I've chosen local, which has an x next to the word to delete it.](../../img/single-component-tutorials-servo-mousemover/servo-config.png)
+
+Click the **Save Config** button.
+
+<div class="td-max-width-on-larger-screens">
+  {{<gif webm_src="../../img/single-component-tutorials-servo-mousemover/save-config.webm" mp4_src="../../img/single-component-tutorials-servo-mousemover/save-config.mp4" alt="Square button that says Save Config with a graphic of an old floppy disk in the corner. A mouse in the shape of a extended finger clicks the button." max-width="200px">}}
+</div>
+
+## Control the servo in Viam app
+
+If everything went well, the servo started to move.
+
+<div class="td-max-width-on-larger-screens">
+  {{<gif webm_src="../../img/single-component-tutorials-servo-mousemover/its-alive.webm" mp4_src="../../img/single-component-tutorials-servo-mousemover/its-alive.mp4" alt="Small square .gif of black and white frankenstein clip with man with brown hair in lab coat frantically looking around and yelling while text that says \"IT'S ALIVE! IT'S ALIVE!\" is on screen. There's a very tall man with something on his head on a table behind him and another man sitting on a chair behind the table." max-width="400px">}}
+</div>
+
+Navigate to the **CONTROL** tab and press the **STOP** button on the servo card (matching what you named the servo) to stop the servo.
+
+<div class="td-max-width-on-larger-screens">
+  {{<gif webm_src="../../img/single-component-tutorials-servo-mousemover/stop.webm" mp4_src="../../img/single-component-tutorials-servo-mousemover/stop.mp4" alt="Red square button with the word STOP on it with an animation of a finger mouse pointer clicking it." max-width="150px">}}
+</div>
+
+Click on the top of the servo card to open the servo controls.
+
+**Angle** will determine what speed your servo goes and in what direction.
```

Comment:
Isn't this just for angle and not speed? i think there is speed/direction outside of angle

Reply:
not for continuous servos